sdks/python/apache_beam/runners/worker/statesampler.pyx

```diff
@@ -190,6 +195,13 @@ cdef class StateSampler(object):
     if self.started and not self.finished:
       self.stop()
 
+  def current_state(self):
+    """Returns the current ScopedState.
+
+    This operation is not thread safe, and should only be used to check, not to
+    update information in the current state."""
+    return self.scoped_states_by_index[self.current_state_index]
```

Comment:
<!--thread_id:cc_158386112_t; commit:3e784cbd5e110c68b56b1b4349510d712afe3b20; resolved:0-->
<!--section:context-quote-->
> **pabloem** wrote:
> My intention is for this method to be used just like the get_info method. Would they not work the same?

<!--section:body-->
If this makes sense, I'll mark as done.